generated/nidaqmx/_base_interpreter.py

```diff
@@ -0,0 +1,1621 @@
+        pass
+
+    def get_ai_chan_cal_exp_date(
+            self, task, channel_name, year, month, day, hour, minute):
+        pass
+
+    def get_analog_power_up_states(
+            self, device_name, channel_name, state, channel_type):
+        pass
+
+    def get_analog_power_up_states_with_output_type(
+            self, channel_names, state_array, channel_type_array, array_size):
+        pass
+
+    def get_arm_start_trig_timestamp_val(self, task, data):
+        pass
+
+    def get_arm_start_trig_trig_when(self, task, data):
+        pass
+
+    def get_auto_configured_cdaq_sync_connections(
+            self, port_list, port_list_size):
+        pass
+
+    def get_buffer_attribute_uint32(self, task, attribute, value):
+        pass
+
+    def get_cal_info_attribute_bool(self, device_name, attribute, value, size):
+        pass
+
+    def get_cal_info_attribute_double(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_cal_info_attribute_string(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_cal_info_attribute_uint32(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_chan_attribute_bool(self, task, channel, attribute, value, size):
+        pass
+
+    def get_chan_attribute_double(self, task, channel, attribute, value, size):
+        pass
+
+    def get_chan_attribute_double_array(
+            self, task, channel, attribute, value, size):
+        pass
+
+    def get_chan_attribute_int32(self, task, channel, attribute, value, size):
+        pass
+
+    def get_chan_attribute_string(self, task, channel, attribute, value, size):
+        pass
+
+    def get_chan_attribute_uint32(self, task, channel, attribute, value, size):
+        pass
+
+    def get_device_attribute_bool(self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_double(self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_double_array(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_int32(self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_int32_array(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_string(self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_uint32(self, device_name, attribute, value, size):
+        pass
+
+    def get_device_attribute_uint32_array(
+            self, device_name, attribute, value, size):
+        pass
+
+    def get_digital_logic_family_power_up_state(
+            self, device_name, logic_family):
+        pass
+
+    def get_digital_power_up_states(self, device_name, channel_name, state):
+        pass
+
+    def get_digital_pull_up_pull_down_states(
+            self, device_name, channel_name, state):
+        pass
```

Comment:
Power-up state functions uses varargs in the C API, so they will require a special case. Think of them as getting/setting either an array of (channel, state) tuples or parallel arrays of channels and states.

I think the way you call them in C is something like this:
```
int32 states[4];
DAQmxGetDigitalPowerUpStates("Dev1", "port0/line0", &states[0], "port0/line1", &states[1], "port0/line2", &states[2], "port0/line3", &states[3], (const char*)NULL);
DAQmxSetDigitalPowerUpStates("Dev1", "port0/line0", states[0], "port0/line1", states[1], "port0/line2", states[2], "port0/line3", states[3], (const char*)NULL);
```